packages/web3-eth/test/integration/defaults.test.ts

```diff
@@ -203,7 +203,7 @@ describe('defaults', () => {
 			expect(web3Eth.defaultBlock).toBe('earliest');
 
 			// set by create new instance
-			eth2 = new Web3Eth({
+			const eth2 = new Web3Eth({
```

Comment:
using local instances so different tests dnt interfere each other's values , and locally close connection instead of closing on last instance at end and other instances dangling